rapyuta_io/clients/device.py

```diff
@@ -272,6 +273,8 @@ def __init__(self, name, runtime=None, runtime_docker=False, runtime_preinstalle
         self._ros_workspace = ros_workspace
         self.description = description
         self.python_version = python_version
+        self.config_variables = config_variables if config_variables else {}
+        self.labels = labels if labels else {}
```

Comment:
suggestion: I think default None value makes the if check redundant, we can directly assign the values here without doing the if check.

Reply:
or you can do

labels = labels or {}